examples/user_defined_iface.yml

```diff
@@ -0,0 +1,43 @@
+---
+user-defined:
+    interface-types:
+        - name: vxcan
+          handler-script: |
+            #!/bin/bash
+            ifname=$1
+            action=$2
+
+            if [ "$action" = "device-add" ]; then
```

Comment:
The action is called "device-add" but we call devices "interfaces" elsewhere. Also, what is the reason to call it action instead of `state` and then using the same values for state as we allow in the list of interfaces?

Reply:
Ah, it is a dispatcher script.